app/graphql/types/digital_object_interface.rb

```diff
@@ -57,14 +57,14 @@ def resources
       end
     end
 
-    def self.title_for(object)
+    def self.title_for(object, indexable = false)
```

Comment:
Do you mind if we're a little more literal with the second param, calling it something like `sortable`?  I found it slightly confusing at first because there isn't a reason why the full title _isn't_ indexable -- we just wouldn't want to index it in a sortable title field.

Reply:
That seems like a good idea. Can you `suggest a change`?

Reply:
Will do!

Reply:
```suggestion
    def self.title_for(object, sortable = false)
```